openstudiocore/src/utilities/core/QJson.cpp

```diff
@@ -1,125 +0,0 @@
-/***********************************************************************************************************************
```

Comment:
@jmarrec is this entire file removed?

Reply:
Yes sir. I removed the Json-portion of Attribute, which isn't being used anywhere (it uses XML for I/O), so this could just go*, and removed from `utilties/CMakeList.txt`  https://github.com/NREL/OpenStudio/pull/3382/files#diff-6f721ba6b2eb2c989d633da86dfddb30

* (even if it couldn't I wouldn't have kept a file named `QJson` here anyways, would have replaced that with `jsoncpp`)